lib/services/service_brokers/v2/schema.rb

```diff
@@ -20,7 +22,12 @@ def to_json
 
     private
 
-    def validate_against_metaschema
+    def validate_metaschema_provided
+      return unless errors.blank?
+      add_schema_error_msg('Schema must have $schema key but was not present') unless @schema['$schema']
```

Comment:
@jenspinney 

Minor nitpick, but I think this error message could be worded a bit more clearly -- maybe something like `Schema missing required $schema key`? 🤔 💭 

Just something to consider if you touch this again, not going to block merging the PR over that.